packages/go/index.ts

```diff
@@ -479,13 +480,20 @@ function isReadable(v: any): v is Readable {
   return v && v.readable === true;
 }
 
-async function copyEntrypoint(entrypoint: string, dest: string): Promise<void> {
-  const data = await readFile(entrypoint, 'utf8');
+async function copyEntrypoint(files: string[], dest: string): Promise<void> {
+  for (const file of files) {
```

Comment:
Can we change this function back so the copyEntrypoint() function can be invoked multiple times? I think line 547 can be changed to a `files.map()` so you run all the copy files concurrently.